v2/tools/generator/internal/astbuilder/declarations.go

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astbuilder
+
+import (
+	"fmt"
+
+	"github.com/dave/dst"
+)
+
+func Declarations(declarations ...any) []dst.Decl {
```

Comment:
Duplicate from your other PR?

Reply:
I pulled it out of here for that PR, knowing I'd need to rebase this one.